src/tests/functional/plugin/shared/include/behavior/plugin/core_integration.hpp

```diff
@@ -282,6 +282,18 @@ TEST(IEClassBasicTest, smoke_SetConfigHeteroThrows) {
                                  CommonTestUtils::DEVICE_HETERO));
 }
 
+TEST_P(IEClassBasicTestP, SetGetConfigForTbbTerminateThrows) {
```

Comment:
do we have test for `GetConfig` and `get_property`? E.g. to check that by default TBB threads are not terminated.

Reply:
Actually, there is no test item to read default value of config/property, the test items contains SetConfig()/set_property() and then GetConfig()/get_property(). It should be necessary to add test cases to check whether the default property/config is false.

Reply:
yes, I am asking about such test